yb-voyager/src/migassessment/sizing.go

```diff
@@ -642,13 +643,13 @@ func getExperimentFile() (string, error) {
 		}
 	}
 	if !fetchedFromRemote {
-		err := os.WriteFile(filepath.Join(AssessmentMetadataDir, EXPERIMENT_DATA_FILENAME), experimentData20240, 0644)
+		err := os.WriteFile(filepath.Join(AssessmentDir, DBS_DIR, EXPERIMENT_DATA_FILENAME), experimentData20240, 0644)
```

Comment:
I see `filepath.Join(AssessmentDir, DBS_DIR, EXPERIMENT_DATA_FILENAME)` getting used repeatedly.

Can we introduce a variable: `ExperimentDBPath` or something